include/map_main.h

```diff
@@ -44,7 +44,11 @@ struct MapOptions {
     bool illumina { false };
     bool clean { false };
     bool binomial { false };
+    bool do_not_auto_update_params { false };
     uint32_t max_covg { 300 };
+    float min_absolute_gene_coverage { 3.0 };
+    float min_relative_gene_coverage { 0.05 };
+    float max_relative_gene_coverage { 100 };
```

Comment:
oh look we define them here too

Reply:
Yeah, actually all CLI defaults are duplicated... will refactor all of them, not just these 3